pandas/util/_decorators.py

```diff
@@ -107,6 +108,16 @@ def _deprecate_kwarg(func):
         @wraps(func)
         def wrapper(*args, **kwargs):
             old_arg_value = kwargs.pop(old_arg_name, None)
+
+            if new_arg_name is None and old_arg_value is not None:
+                msg = (
+                    "the '{old_name}' keyword is no longer supported"
```

Comment:
the {old_name} keyword is deprecated and will be removed in a future version

 is our typical phrasing here.

Reply:
Thank you.  Please refer to 699805198f7e31e3f6787b612ec507dd71c056a6